app/renderer/components/.postcssrc

```diff
@@ -0,0 +1,3 @@
+{
+  "modules": true
+}
```

Comment:
This tells Parcel that any CSS file inside `app/renderer/components` is loaded using CSS modules... anything outside of this folder is just raw CSS.